### DISCUSSION TITLE
Connection Error Loading nspanel_us.tft

### DISCUSSION THREAD

**Discussion by jsilveira-ebsco** [Blackymas/NSPanel_HA_Blueprint#640] [Q&A]
Hi all, I'm hoping someone might have some insight and be able to help with  [Issue #539](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/539) (Connection Error Loading nspanel_us.tft) that I posted a couple of weeks ago.  
- I've read [v.3.1.0 - New features and US Version](https://github.com/Blackymas/NSPanel_HA_Blueprint/releases/tag/v.3.1.0) and have tried everything I’ve read regarding Issues [#166](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/166), [#177](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/177), and [#197](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/197)
- Although I did get some feedback (thank you [deejaybeam](https://github.com/deejaybeam)), I haven't yet been able to get me over the hump, but still am hopeful

Any help that anyone has time for and is generous enough to offer would be very much appreciated

**Comment by edwardtfn** [comment]
More details on #539
